clang/test/InstallAPI/objcclasses.test

```diff
@@ -0,0 +1,86 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: sed -e "s|DSTROOT|%/t|g" %t/inputs.json.in > %t/inputs.json
+
+/// Check multiple targets are captured.
```

Comment:
What do you mean with multiple targets here?

Reply:
Thanks for calling this out, forgot to remove.